lib/display/progress_bar.ex

```diff
@@ -4,14 +4,19 @@ defmodule Display.ProgressBar do
 
   def progress_bar(%{current: current, total: total}) do
     arrow = calculate_progress(current, total) |> build_arrow
+    progress_percentage = calculate_percentage(current, total)
 
-    "|" <> String.pad_trailing(arrow, @progress_bar_length) <> "| #{current} of #{total}"
+    "|" <> String.pad_trailing(arrow, @progress_bar_length) <> "| #{current} of #{total} -> #{progress_percentage}% complete"
   end
 
   defp calculate_progress(current, total) do
     round(current / total * @progress_bar_length)
   end
 
+  defp calculate_percentage(current, total) do
+    Float.round(current / total * 100, 1)
```

Comment:
I wonder if this would look better without any decimal places?
```
0 of 12 -> 0% complete
```

Reply:
I thought it  would be useful for people like me who wants to know the progress a little more precisely. But either way it's fine doesn't matter too much.

Reply:
I don't have a strong opinion. I tend to agree that decimals aren't terribly useful in this context.